src/pages/CirclesPage/CirclesPage.tsx

```diff
@@ -231,6 +265,8 @@ const CircleRow = ({ circle, onButtonClick }: CircleRowProps) => {
           '.circle-row-menu-indicator': { display: 'none' },
         },
         ...(nonMember ? nonMemberPanelCss : { cursor: 'pointer' }),
+        transition: 'opacity 300ms ease-in-out',
```

Comment:
Nice.

The borders are missing from the non-member circle. please add a transparent border to this panel css block, after line 261:
`border: '1px solid transparent',`

That way all the panels have a border and the non member panel will get its correct style as defined here:
```
const nonMemberPanelCss: CSS = {
  backgroundColor: '$background',
  borderColor: '$borderMedium',
};
```

Reply:
I went ahead and made this change and also addressed another issue with circleRow coming from `main`, as circle row is also being used on the new Org page, and the state prop isn't used there, so I made it optional